types/office-js-preview/index.d.ts

```diff
@@ -325,7 +325,47 @@ declare namespace Office {
      * @param useShortNamespace True to use the shortcut alias; otherwise false to disable it. The default is true.
      */
     function useShortNamespace(useShortNamespace: boolean): void;
+    /**
+     * Represents the add-in.
+     */
+    const addin: Addin;
+    /**
+     * Represents the ribbon associated with the Office application.
+     */
+    const ribbon: Ribbon;
+    /**
+     * Check if the specified requirement set is supported by the host Office application.
+     * @param name - Set name; e.g., "MatrixBindings".
+     * @param minVersion - The minimum required version; e.g., "1.4".
+     */
+    function isSetSupported(name: string, minVersion?: string): boolean;
     // Enumerations
+    /**
+     * Startup behavior of the add-in.
```

Comment:
Can we rephrase this to describe what's happening rather than repeating the enum name? Like, this is the behavior of the add-in when the Office host starts or when the add-in is opened?

Reply:
done